src/compas/geometry/primitives/polyline.py

```diff
@@ -412,6 +431,98 @@ def divide_polyline_by_length(self, length, strict=True, tol=1e-06):
 
     divide_by_length = divide_polyline_by_length
 
+    def split_polyline_by_length(self, length, strict=True):
```

Comment:
what is the difference between split and divide?

Reply:
The difference is that divide returns a list of the points that the polyline is divided into. Split returns a list of Polylines. It breaks the polyline into different parts, without disregarding it's geometry.